File: 01_Language/01_Functions/js/getdate.js
function getdate(timestamp) {
    let d = typeof timestamp === 'undefined' ? new Date() : new Date(timestamp * 1000)

    let _w = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
    let _m = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']

    let res = {}
    res.seconds = d.getSeconds()
    res.minutes = d.getMinutes()
    res.hours = d.getHours()
    res.mday = d.getDate()
    res.wday = d.getDay()
    res.mon = d.getMonth() + 1
    res.year = d.getFullYear()
    res.yday = Math.floor((d - (new Date(d.getFullYear(), 0, 1))) / 86400000)
    res.weekday = _w[d.getDay()]
    res.month = _m[d.getMonth()]
    res[0] = parseInt(d.getTime() / 1000)
    return res
}

console.log(getdate())

File: 01_Language/01_Functions/python/getdate.py
# coding: utf-8

import time

_MONTHNAMES = [None, "January", "February", "March", "April", "May", "June",
               "July", "August", "September", "October", "November", "December"]
_DAYNAMES = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]


def getdate(timestamp=None):
    if timestamp is None:
        timestamp = time.time()
    t = time.localtime(timestamp)

    res = {}
    res["seconds"] = t.tm_sec
    res["minutes"] = t.tm_min
    res["hours"] = t.tm_hour
    res["mday"] = t.tm_mday
    res["wday"] = t.tm_wday
    res["mon"] = t.tm_mon
    res["year"] = t.tm_year
    res["yday"] = t.tm_yday - 1
    res["weekday"] = _DAYNAMES[t.tm_wday]
    res["month"] = _MONTHNAMES[t.tm_mon]
    res[0] = timestamp
    return res


if __name__ == '__main__':
    print(getdate())
